examples/policies/rate_limiting_to_service_configuration.lua

```diff
@@ -0,0 +1,31 @@
+local policy_chain = require('apicast.policy_chain').default()
+
+local rate_limit_policy = require('apicast.policy.rate_limiting_to_service').new({
```

Comment:
I think this policy can have way simpler name than `rate_limiting_to_service`. Just `rate_limit` should be enough.